GrandCentralBoard/Widgets/Watch/Source/GoogleCalendarNameSource.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  GoogleCalendarNameSource.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 12.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import GrandCentralBoardCore
+
+final class GoogleCalendarNameSource : CalendarNameSource {
+
+    private let dataProvider: CalendarDataProviding
+    private let calendarID: String
+
+    init(calendarID: String, dataProvider: CalendarDataProviding) {
+        self.dataProvider = dataProvider
+        self.calendarID = calendarID
+    }
+
+    override func read(closure: (ResultType) -> Void) {
```

Comment:
Note: One more case in which code guidelines would be helpful - we use `completionBlock` and `closure` names for the same thing in different places.